### ISSUE TITLE
Do not treat subsequent actions as dangling assets 

### ISSUE THREAD

**Issue by zdne** [apiaryio/snowcrash#200]
Dangling assets algorithm is too greedy and processes multiline payload signature with action definition as a dangling assets. 

The root cause is obviously the Markdown syntax / parser but we can walk the extra step towards user-friendliness and detect these cases...

### Given

```
# /
### PUT
+ Response 401
### GET
+ Resposne 200
```

### Output

```json
OK.
warning: (10)  message-body asset is expected to be a pre-formatted code block, separate it by a newline and indent every of its line by 8 spaces or 2 tabs :14:21
warning: (5)  ignoring unrecognized block :35:14

_version: 2.0
metadata:
name:
description:
resourceGroups:
- name:
  description:
  resources:
  - name:
    description:
    uriTemplate: "/"
    model:
    parameters:
    actions:
    - name:
      description:
      method: "PUT"
      parameters:
      examples:
      - name:
        description:
        requests:
        responses:
        - name: "401"
          description:
          headers:
          body: "### GET\n\n"
          schema:
```

**Comment by pksunkara** [comment]
@kylef Why is this linked in 3.1.0-pre.0 drafter release?

**Comment by kylef** [comment]
@pksunkara Because you said it was fixed by your pull request (here: https://github.com/apiaryio/snowcrash/pull/395#issuecomment-231416313)

**Comment by pksunkara** [comment]
It was supposed to fix this. But looks like it didn't.

**Comment by pksunkara** [comment]
Fixed by [Drafter 3.1.0](https://github.com/apiaryio/drafter/releases/tag/v3.1.0)
